build/isnearupgrade.go

```diff
@@ -5,5 +5,5 @@ import (
 )
 
 func IsNearUpgrade(epoch, upgradeEpoch abi.ChainEpoch) bool {
-	return epoch > upgradeEpoch-Finality && epoch < upgradeEpoch+Finality
+	return upgradeEpoch >= 0 && epoch > upgradeEpoch-Finality && epoch < upgradeEpoch+Finality
```

Comment:
This is probably easier to read: 

```suggestion
	if upgradeEpoch < 0 {
	    return false
	}
	return epoch > upgradeEpoch-Finality && epoch < upgradeEpoch+Finality
```

Reply:
fair enough